_posts/en/pages/2017-01-01-download.md

```diff
@@ -56,11 +56,15 @@ releasekeys: "Bitcoin Core Release Signing Keys"
 
 pgp_key_fingerprint: "PGP key fingerprint"
 verify_download: "Verify your download"
-verification_recommended: "Download verification is optional but highly recommended.  Click one of the lines below to view verification instructions for that platform."
+verification_recommended: >
+  Download verification is optional but highly recommended.
+  Click one of the lines below to view verification instructions for that platform.
+  Note that due to a <a href=\"https://bugs.launchpad.net/snapstore/+bug/1803914\">series</a>
+  <a href=\"https://bugs.launchpad.net/launchpad/+bug/1331914\">of</a> mishandlings of security issues
+  by Canonical/Ubuntu, use of Bitcoin wallets on Ubuntu is strongly discouraged.
```

Comment:
Why is this "use of Bitcoin wallets on Ubuntu is strongly discouraged" rather than "installing Bitcoin through a PPA is strongly discouraged"? Is there an issue with users downloading the binary from bitcoincore.org and running it on ubuntu?

Reply:
Its not the PPA. The issue is that Ubuntu tells their users to go download packages from random people and refuses to do anything about the major security implications that has.